PrismSampleApp/PrismSampleApp/ViewModels/ViewModelBase.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Prism.Commands;
+using Prism.Mvvm;
+using Prism.Navigation;
+using PrismSampleApp.Model;
+using PrismSampleApp.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PrismSampleApp.ViewModels
+{
+    public class ViewModelBase : BindableBase, IInitialize, INavigationAware, IDestructible,IWebApiService
+    {
+        protected INavigationService NavigationService { get; private set; }
+
+
+        public List<Result> RecievedContacts { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
Why RecievedContacts in ViewmodelBase?

Reply:
removed RecievedContacts from ViewModelBase